docs/developer-docs/integrations/sns/launching/launch-summary-1proposal.md

```diff
@@ -0,0 +1,500 @@
+* NNS root has been added as a co-controller of the dapp to be decentralized (Stage 3
+was executed succesfully)
+* the SNS-W canister has sufficient cycles.
+
+If these conditions are not met, the proposal will fail immediately and the control is 
+given back exclusively to the original dapp developers.
+
+If successful, at the end of this stage the proposal is adopted and the conditions are met.
+This means that we are in the following situation:
+
+#### Table 2: NNS Proposals
+ <table>
+  <tr>
+    <th>NNS Proposal</th>
+    <th>State</th>
+  </tr>
+  <tr>
+    <td>SNS creation proposal</td>
+    <td class="light-orange-text">Adopted</td>
+  </tr>
+</table>
+
+
+### 5. (Automatically) SNS-W deploys SNS canisters
+As a first step of the automatically triggered
+SNS creation, [SNS-W](../introduction/sns-architecture.md#SNS-W)
+creates SNS canisters on the SNS subnet.
+At this stage the SNS canisters are not yet initialized with any meaningful values.
+
+This results in the following situation:
+
+#### Table 1: Canisters
+<table>
+  <tr>
+    <th>Canisters</th>
+    <th>Subnet</th>
+    <th>Controlled by</th>
+    <th>State</th>
+  </tr>
+  <tr>
+    <td>dapp canister(s)</td>
+    <td>application subnet</td>
+    <td>dapp developer principal, NNS root</td>
+    <td>operational</td>
+  </tr>
+  <tr>
+    <td class="light-green-text">SNS swap</td>
+    <td class="light-green-text">SNS subnet</td>
+    <td class="light-green-text">NNS root</td>
+    <td class="light-green-text">not initialized</td>
+  </tr>
+ <tr>
+    <td class="light-green-text">SNS governance</td>
+    <td class="light-green-text">SNS subnet</td>
+    <td class="light-green-text">SNS root</td>
+    <td class="light-green-text">not initialized</td>
+  </tr>
+ <tr>
+    <td class="light-green-text">SNS ledger</td>
+    <td class="light-green-text">SNS subnet</td>
+    <td class="light-green-text">SNS root</td>
+    <td class="light-green-text">not initialized</td>
+  </tr>
+ <tr>
+    <td class="light-green-text">SNS root</td>
+    <td class="light-green-text">SNS subnet</td>
+    <td class="light-green-text">SNS governance</td>
+    <td class="light-green-text">not initialized</td>
+  </tr>
+</table>
+
+
+### 6. (Automatically) SNS-W sets SNS root as single controller of dapp
+Once the SNS canisters are deployed, [SNS-W](../introduction/sns-architecture.md#SNS-W)
+sets the SNS root as the single controller of the dapp canister(s).
+
+For technical reasons, the NNS root canister was added as the co-controller of the dapp
+in Stage 3.
+Therefore, the SNS-W orchestrates the necessary updates involving NNS root for 
+making the appropriate changes.
+
+In detail, this includes two steps:
+* First, SNS-W adds the newly created SNS root cansiter as the dapp canister(s)
+  controller and removes the original dapp developers
+  This is done via a call to NNS root who is the co-controller of the dapp cansiter(s)
+  and thus has the necessary permissions.
```

Comment:
If you want to be 100% accurate, I'd suggest writing this

```suggestion
* First, SNS-W removes the original dapp developers from controlling the dapp(s). Next SNS-W adds the newly created SNS root canister as the dapp canister(s) controller. This is done via a call to NNS root who is the co-controller of the dapp canister(s) and thus has the necessary permissions.
```

Reply:
yes. let's be precise.
will add in next commit.